### ISSUE TITLE
False positive in `Rails/FindEach` with `Capybara`

### ISSUE THREAD

**Issue by tobypinder** [rubocop-hq/rubocop#4519]
`Rails/FindEach` picks up on [`Capybara`](https://github.com/teamcapybara/capybara) node traversal as though it were ActiveRecord statements and suggests `find_each` as a nonexisting alternative to `find`.

Details below, but I'm not sure what I'd recommend as a fix - perhaps

## Expected behavior

`Rails/FindEach` should only detect `ActiveRecord` specific violations.

## Actual behavior

`Rails/FindEach` suggests violation that would break `Capybara` code.

## Steps to reproduce the problem

Any `Capybara` code using `all` followed by `each`, eg

```ruby
page.all('.row').each do |row|
  do_stuff(row)
end
```

## RuboCop version

```
➜  rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-linux)
```

**Comment by mikegee** [comment]
> I'm not sure what I'd recommend as a fix

Me either. How can Rubocop differentiate ActiveRecord from Capybara? It's just doing static analysis. 

Maybe exclude your `test` / `spec` for `Rails/FindEach`?

**Comment by Drenmi** [comment]
Excluding `spec/features/**/*` from inspection seems like the right call. It should possibly even be excluded by default.

```yaml
Rails/FindEach:
  Exclude:
    - "spec/features/**/*"
```

**Comment by tobypinder** [comment]
@drenmi that would seem to be a sensible approach, I'd advocate for that to be a default as well. If there's any way to also highlight the Cop's limitations in the documentation, that would also be great for people who end up using Capybara elsewhere in their projects, and as such need to add additional `exclude` paths or go without the cop entirely.

**Comment by bbatsov** [comment]
I'm closing this ticket due to no recent activity. Feel free to re-open it if you ever come back to it.

**Comment by tobypinder** [comment]
@bbatsov Fair enough. The mitigation of excluding specs from `Rails/FindEach` works well enough for us, I would still wonder if that should be a default. That said I can't attest to Rubocop's current behaviour here since we have just gone with our existing config since reporting this.
